app/scripts/lib/ppom/security-alerts-api.ts

```diff
@@ -1,22 +1,30 @@
-import { Hex } from '@metamask/utils';
+import { Hex, JsonRpcRequest } from '@metamask/utils';
 import { SecurityAlertResponse } from './types';
 
 const ENDPOINT_VALIDATE = 'validate';
 const ENDPOINT_SUPPORTED_CHAINS = 'supportedChains';
 
-export type SecurityAlertsAPIRequest = {
+type SecurityAlertsAPIRequestBody = {
   method: string;
   params: unknown[];
 };
 
+export type SecurityAlertsAPIRequest = Omit<
+  JsonRpcRequest,
+  'method' | 'params'
+> &
+  SecurityAlertsAPIRequestBody;
```

Comment:
Defines expected type for `JsonRpcRequest` objects that are compatible with the security alerts API.